src/client.cpp

```diff
@@ -523,6 +531,20 @@ GAME_ERROR CGameLibRetro::GetRichPresenceEvaluation(char* evaluation, size_t siz
   return GAME_ERROR_NO_ERROR;
 }
 
+GAME_ERROR CGameLibRetro::ActivateAchievement(unsigned cheevo_id, const char* memaddr)
+{
+  CCheevos::Get().ActivateAchievement(cheevo_id, memaddr);
+
+  return GAME_ERROR_NO_ERROR;
+}
+
+GAME_ERROR CGameLibRetro::GetCheevo_URL_ID(void (*Callback)(const char* achievement_url,
+                                                            unsigned cheevo_id))
+{
+  CCheevos::Get().GetCheevo_URL_ID(Callback);
```

Comment:
Same here, if `GetCheevo_URL_ID` fails, we never know it because it returns `void`. Again, if it's not essential to handle errors around this for now, it's fine.